### ISSUE TITLE
Benchmark against competition

### ISSUE THREAD

**Issue by mre** [tinysearch/tinysearch#136]
People are interested in a size-comparison between
* tinysearch
* lunr.js
* elasticlunr
* flexsearch
* fuse.js

If anyone wants to do a comparison, post a comment here.
Some ideas:
* Use an open dataset, e.g. Shakespeare texts https://shakespeare.folger.edu/download/
* Create a size histogram of the results that we can include into the README.
* Add the code to the repository under a `bench` folder and create a pull request. This will make the benchmark reproducible in the future.

Feel free to ask questions here before starting.

**Comment by Jieiku** [comment]
Awesome idea. I currently use Zola for my blog, it seems to come with elasticlunr, I am in the process of trying to evaluate which solution is lightest.

**Comment by Jieiku** [comment]
I found that somebody actually made one that tests a lot of them!

https://nextapps-de.github.io/flexsearch/bench/

Unfortunately tinysearch is not in their benchmark list.

**Comment by mre** [comment]
Interesting. We should add it there; also Stork and pagefind.

**Comment by Jieiku** [comment]
one thing I dont see in their benchmark is bandwidth used... so they really should have a second benchmark for that.

**Comment by mre** [comment]
How exactly do you define bandwidth? Queries per second?
You could create an issue for the project to discuss that.

**Comment by Jieiku** [comment]
no, what i mean is hit the website. perform a search, how many KB did it take to download the search javascript + wasm + index to perform that search.

For instance Stork is around 700kb whereas tinysearch is around 100kb for abridge.

Obviously the amount of content will affect the index size, in their test they appear to be using Gulliver's Travels

**Comment by mre** [comment]
Gotcha. Yeah, that would indeed be a great comparison.
